.pipelines/templates/stages/azl_installer/validate-usb-iso.yml

```diff
@@ -37,21 +37,22 @@ steps:
       uri_default = "qemu:///system"
       EOF
 
-      cp $(Pipeline.Workspace)/azl-installer/azl-installer.iso /tmp/azl-installer.iso
+      cp $(Pipeline.Workspace)/azl-installer/${ISO_NAME}.iso /tmp/azl-installer.iso
```

Comment:
any reason that ISO_NAME isn't a parameter?  might be clearer to trace where setting comes from as param.